databases/models/User.js

```diff
@@ -71,22 +71,30 @@ module.exports = (sequelize, DataTypes) => {
       status: { type: DataTypes.BOOLEAN, allowNull: false },
       bio: { type: DataTypes.TEXT, allowNull: true },
       is_banned: {
-        type: DataTypes.BOOLEAN, 
+        type: DataTypes.BOOLEAN,
         defaultValue: false,
       },
       is_anonymous: {
-        type: DataTypes.BOOLEAN, 
+        type: DataTypes.BOOLEAN,
         defaultValue: false,
       },
       encrypted: {
         type: DataTypes.STRING,
-        allowNull: true
-      }
+        allowNull: true,
+      },
+      allowAnonDm: {
+        type: DataTypes.BOOLEAN,
+        defaultValue: true,
+      },
+      onlyReceivedAnonDmFromUserFollowing: {
+        type: DataTypes.BOOLEAN,
+        defaultValue: false,
+      },
```

Comment:
<img width="714" alt="image" src="https://github.com/BetterSocial/user-api/assets/55779793/aad7cd76-4171-4179-aee9-7b0de77f9ba1">

mas @falhar this error occured when changing anon dm setting on this endpoint
PATCH {{baseUrl}}/profiles/settings

Here are my alternatives solution:
1. Changing column name on admin-system migration to lowercase with snake case (allow_anon_dm and only_received_...) OR
2. giving column name to both of this field
`allowAnonDm: {
...,
field: 'allowAnonDm'
}`